cprover_bindings/src/goto_program/typ.rs

```diff
@@ -286,6 +289,7 @@ impl Type {
             CInteger(CIntType::Bool) => Some(mm.bool_width),
             CInteger(CIntType::Char) => Some(mm.char_width),
             CInteger(CIntType::Int) => Some(mm.int_width),
+            Integer => Some(mm.int_width),
```

Comment:
This one is unbounded, it doesn’t have a bitwidth.

Reply:
Done. Thanks!

Reply:
Actually, do you think we should return `None` or just `Some(x)` where x is a large number?

Reply:
Maybe we can have an "infinite" token? But `Some(x)` is semantically wrong for all `x`